app/models/course/assessment/answer/programming.rb

```diff
@@ -43,4 +43,17 @@ def attempting_times_left
   def grade_inline?
     false
   end
+
+  def downloadable?
```

Comment:
The other thing is, should we set this at the `question` model instead? It seems to be a property of the question, not the answer.